halpybot/commands/edsm.py

```diff
@@ -65,23 +131,27 @@ async def cmd_distlookup(ctx: Context, args: List[str], cache_override):
     """
     Check EDSM for the distance between two known points.
 
-    Usage: !distance <--new> [system/cmdr 1] : [system/cmdr 2]
+    Usage: !distance <--new> [System/CMDR/caseID 1] : [System/CMDR/caseID 2] : <Jump Range>
     Aliases: dist
     """
     try:
-        pointa, pointb = [
-            "".join(point).strip() for point in "".join(args).split(":", 1)
-        ]
-    except (ValueError, IndexError):
-        return await ctx.reply("Please provide two points to look up, separated by a :")
-    if not pointb:
-        return await ctx.reply("Please provide two points to look up, separated by a :")
+        points = await differentiate(ctx=ctx, args=args)  # Process provided arguments
+    except DifferentiateArgsIssue:
+        # Arguments were malformed, user has already been informed, abort
+        return
+
     distance, direction = await checkdistance(
-        pointa, pointb, cache_override=cache_override
+        points[0][0], points[1][0], cache_override=cache_override
     )
+    if len(points) < 3:  # No Jump Range given, respond with Distance
+        return await ctx.reply(
+            f"{points[0][1]} is {distance} LY {direction} of " f"{points[1][1]}."
```

Comment:
this points`[x][y]` rubs me the wrong way and is not readable.
consider refactoring.